src/substrate/storageFetcher.ts

```diff
@@ -246,6 +250,35 @@ export class StorageFetcher extends IStorageFetcher<ApiPromise> {
     return proposedEvents.map((data) => ({ blockNumber, data }));
   }
 
+  public async fetchTreasuryBounties(blockNumber: number): Promise<CWEvent<ITreasuryBountyProposed>[]> {
+    log.info('Migrating treasury bounties...');
+    const approvals = await this._api.query.treasury.bountyApprovals();
+    const nBounties = await this._api.query.treasury.bountyCount();
+    const bountyIds: number[] = [];
+
+    for (let i = 0; i < +nBounties; i++) {
+      if (!approvals.some((id) => +id === i)) {
+        bountyIds.push(i);
+      }
+    }
+    const bounties = await this._api.query.treasury.bounties.multi<Option<Bounty>>(bountyIds);
+    const proposedEvents = bountyIds.map((id, index) => {
+      if (!bounties[index] || !bounties[index].isSome) return null;
+      const { proposer, value, fee, curatorDeposit, bond, } = bounties[index].unwrap();
+      return {
+        kind: EventKind.TreasuryBountyProposed,
+        bountyIndex: +id,
+        proposer: proposer.toString(),
+        value: value.toString(),
+        fee: fee.toString(),
+        curatorDeposit: curatorDeposit.toString(),
+        bond: bond.toString(),
+    } as ITreasuryBountyProposed;
```

Comment:
We should generate additional update events here if the bounty is in a state beyond "proposed".